Mu2eG4/geom/ExtShieldUpstream_v07.txt

```diff
@@ -0,0 +1,207 @@
+// Original author David Norvil Brown, University of Louisville
```

Comment:
Can you provide a diff of v07 against v06?

Reply:
I disabled line 158 and enabled line 159